tailwind.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
         themeGray: '#121212',
         primary: '#3ed6f0',
         secondary: '#252e3e',
+        rgbValuesSecondary: 'rbg(37, 46, 62, 0.75)',
```

Comment:
```suggestion
        secondary: 'rbga(37, 46, 62, 0.75)',
```

Let's ditch hex & simplify the theme.

Reply:
you want me to remove both secondary hex and rgbValuesSecondary and only keep secondary as rgb color right ?

Reply:
done